CODEOWNERS

```diff
@@ -10,6 +10,7 @@ daemon/prefilter.go @cilium/bpf
 
 # Agent -> Kernel interaction
 bpf/*.sh @cilium/bpf-loader
+daemon/bpf.sha @cilium/bpf-loader
```

Comment:
Given that `bpf.sha` is modified on datapath changes, it should be reviewed by the datapath team I think

Reply:
@tgraf as well the `bpf/*.sh @cilium/bpf-loader`?

Reply:
`bpf/*.sh` is fine as-is, the shell script is loader territory. It will mean that any changs to `bpf/*.sh` needs approval from the datapath team as it will bump `bpf.sha`. Seems like that's exactly what we want.

Reply:
@tgraf oh right, I forgot the bpf.sha was only including the `bpf/*.sh`